glue_statistics/__init__.py

```diff
@@ -0,0 +1,20 @@
+import os 
+
+REFRESH_LOGO = os.path.abspath(os.path.join(os.path.dirname(__file__), 'glue_refresh.png'))
```

Comment:
Do you have SVG versions of the logos? This would be preferable instead of PNG if possible.